src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
```

Comment:
Ensure that the image path '../images/star.svg' is correct and the image is available at this location. Otherwise, the star icons might not display correctly.